h1st/model/remote_model.py

```diff
@@ -0,0 +1,157 @@
+from typing import Dict, Tuple
+import requests
+from .predictive_model import PredictiveModel
+import pandas as pd
+
+
+class RemoteModel(PredictiveModel):
+    """
+    Class for making inference calls with models served via the Aitomatic Wed API
+
+    Ex: 
+    model = RemoteModel(API_TOKEN, "MyModelName").load_params()
+    predictions = model.predict({'X': MyDataFrame})
+    """
+    PREDICTION_ENDPOINT = ''
```

Comment:
temporary the endpoint will be
- `https://model-api-dev.platform.aitomatic.com/fuzzy/predict` for fuzzy model
- `https://model-api-dev.platform.aitomatic.com/oracle/predict` for oracle model
After the mvp, we will unify the endpoint for any model based on the model type

Reply:
@phamhoangtuan is there a reason this can't be unified before the MVP? Can you point me to the code for the API container?

Reply:
the API is unified to `https://model-api-dev.platform.aitomatic.com/inferencing` for both fuzzy and oracle.
The body for this API are:
  - model_name
  - model_version
  - access_token (We need to put access token to body as can't read it from request headers for now)
  - input_data (json string converted from a dataframe or s3 path)

Reply:
access_token can be put to Header now